src/rules/commentFormatRule.ts

```diff
@@ -39,20 +39,25 @@ export class Rule extends Lint.Rules.AbstractRule {
         options: {
             type: "array",
             items: {
-                type: "string",
-                enum: ["check-space", "check-lowercase", "check-uppercase"],
+                oneOf: [{
+                    type: "string",
+                    enum: ["check-space", "check-lowercase", "check-uppercase"],
+                }, {
+                    type: "array",
```

Comment:
I've checked list of ESLint rules and they use RE to provide patterns, however they use object with "ignorePattern" key to provide it.

As alternative approach I can implement exception argument as object with two possible keys `ignoreWords` - array of strings to ignore and `ignorePattern` - string with RE to ignore, instead of array of words.